src/Sylius/Behat/Page/Admin/Shipment/IndexPage.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Page\Admin\Shipment;
+
+use Behat\Mink\Element\NodeElement;
+use Sylius\Behat\Page\Admin\Crud\IndexPage as BaseIndexPage;
+
+final class IndexPage extends BaseIndexPage implements IndexPageInterface
```

Comment:
I think the page object should not be `final`, it's a common case to extend them